src/init.cpp

```diff
@@ -208,6 +208,7 @@ void DynamicDataLoader::initialize()
     add( "scenario", &scenario::load_scenario );
     add( "start_location", &start_location::load_location );
     add( "skill_boost", &skill_boost::load_boost );
+    add( "ENCHANTMENT", &enchantment::load_enchantment );
```

Comment:
Any particular reason this one is in all-caps?  That doesn't seem to be the norm.

Reply:
no, not really. changed it to lowercase.